charts/agent/values.yaml

```diff
@@ -120,6 +120,7 @@ serviceAccount:
   name: null
 daemonset:
   deploy: true
+  minReadySeconds: 1
```

Comment:
@aroberts87 @irozzo-1A @mavimo Setting 1 will change the default behaviour for everyone.
With this in place every update batch will start only once all the pods in the previous one will reach the `Ready` state. I'm not sure if we want this to become the default behaviour from now on. Thoughts?

Reply:
@AlbertoBarba IMO we should first clarify if this is this is the expected behavior. I don't have the impression from what I see in the docs and the code I would expect that when keeping the default (0), we should be waiting for readiness anyway.
 https://github.com/kubernetes/kubernetes/blob/f39ece24b27062a8d9eb791ffebc252ea1449734/staging/src/k8s.io/api/apps/v1/types.go#L246-L250
https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/#performing-a-rolling-update

Reply:
I agree with @irozzo-1A here. Waiting 1 second after the pod claims ready state shouldn't gain much over just waiting for the pod to be in ready state. The fact that the one second difference here seems to resolve the issue smells to me like there is a bug somewhere else in the system.